components/automate-cli/cmd/chef-automate/summary.go

```diff
@@ -0,0 +1,418 @@
+package main
+
+import (
+	"container/list"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strconv"
+	"time"
+
+	"github.com/chef/automate/components/automate-cli/pkg/status"
+	"github.com/chef/automate/lib/stringutils"
+	"github.com/jedib0t/go-pretty/v5/table"
+	"github.com/spf13/cobra"
+)
+
+var (
+	habUrl                = "https://localhost:9631"
+	clusterOSHealth       = "http://localhost:10144/_cluster/health"
+	a2haHabitatAutoTfvars = terraformPath + "/a2ha_habitat.auto.tfvars"
+	nowFunc               = time.Now
+)
+
+type StatusSummary interface {
+	Run(sshUtil SSHUtil) error
+	FEDisplay() string
+	BEDisplay() string
+}
+
+type Summary struct {
+	feStatus              FeStatus
+	beStatus              BeStatus
+	timeout               int64
+	spinnerTimeout        time.Duration
+	infra                 *AutomteHAInfraDetails
+	statusSummaryCmdFlags *StatusSummaryCmdFlags
+}
+
+type A2haHabitatAutoTfvars struct {
+	HabSupHttpGatewayAuthToken string `json:"hab_sup_http_gateway_auth_token"`
+	HabSupRingKey              string `json:"hab_sup_ring_key"`
+}
+
+// Display Frontend Status
+func (ss *Summary) FEDisplay() string {
```

Comment:
I think instead of FEDisplay() we could use something like FEStatusDisplay() or ShowFEStatus()

Reply:
made changes